README.md

```diff
@@ -2,58 +2,139 @@
 [![codecov](https://codecov.io/gh/transifex/transifex-python/branch/master/graph/badge.svg)](https://codecov.io/gh/transifex/transifex-python)
 
 # Transifex Python Toolkit
-
-Transifex Python Toolkit (referred as **the Toolkit** for brevity) is a collection of tools that allow you to easily localize your applications using Transifex (TX). The toolkit features fetching translations over the air (OTA) to your apps.
+- [Transifex Python Toolkit](#transifex-python-toolkit)
+  * [Overview](#overview)
+    + [Introduction](#introduction)
+      - [Usage Scenarios](#usage-scenarios)
+    + [Features](#features)
+    + [Authentication](#authentication)
+    + [Hosting Translations](#hosting-translations)
+  * [Supported Frameworks](#supported-frameworks)
+    + [Django](#django)
+      - [Installation](#installation)
+      - [Configuration](#configuration)
+      - [Quickstart](#quickstart)
+        * [Add Translation Hooks](#add-translation-hooks)
+        * [Push Source Content to Transifex](#push-source-content-to-transifex)
+        * [Translate Content on Transifex](#translate-content-on-transifex)
+        * [Display Translated Content](#display-translated-content)
+      - [Use in Templates and Views](#use-in-templates-and-views)
+        * [Internationalization in template code](#internationalization-in-template-code)
+        * [Internationalization in Python Code](#internationalization-in-python-code)
```

Comment:
This section was removed, the link points to nowhere

Reply:
Done, the section existed but was not copied, so restored